units/sdl2_mixer.pas

```diff
@@ -117,20 +118,11 @@ procedure Mix_Quit() cdecl; external MIX_LibName {$IFDEF DELPHI} {$IFDEF MACOS}
 
   {* Good default values for a PC soundcard *}
 const
-  MIX_DEFAULT_FREQUENCY = 22050;
+  MIX_DEFAULT_FREQUENCY = 44100;
+  MIX_DEFAULT_FORMAT   = AUDIO_S16SYS;
```

Comment:
`AUDIO_S16SYS` is hidden behind `{$IFDEF FPC}` inside `sdlaudio.inc`, so this should be as well.